src/Web/Rsp.IrasPortal.Web/Areas/Admin/Controllers/UsersController.cs

```diff
@@ -60,7 +60,7 @@ public async Task<IActionResult> Index()
     /// Displays the empty UserView to create a user
     /// </summary>
     [HttpGet]
-    public IActionResult CreateUser()
+    public IActionResult CreateUser() //admin:createuser
```

Comment:
Remove this comment please
```suggestion
    public IActionResult CreateUser()
```